nodejs/listeners/actions/__tests__/__utils__/action-test-util-funcs.js

```diff
@@ -10,6 +10,7 @@ const mockActionCallbackInput = (actionPayload) => ({
     },
     chat: {
       update: global.chatUpdateMockFunc,
+      deleteScheduledMessage: global.deleteScheduledMessageMockFunc,
```

Comment:
I don't know why this mock object had not been necessary but anyway, without this, the test does not pass.